### DISCUSSION TITLE
Symmetries affecting pole figure reconstruction from ODF?

### DISCUSSION THREAD

**Discussion by RammyCa** [mtex-toolbox/mtex#1862] [Ask Anything]
Hi, I am an undergrad working for some summer research, and I stumbled upon trying to reproduce the original pole figures I used for an ODF and it didn't return the same things. I am about to try ghosting correction, although I doubt that is the problem, but I did try first with changing the sample symmetry from an arbitrarily chosen 1 to 622, since that should be the correct symmetry for my geometry (I have Zircon sample data taken at a synchrotron). When trying again to reconstruct the pole figures, it only gives me sections of it that sort of look like inverse pole figures but that's not what I was looking for. What can I do to solve this issue? What does that mean? What other things may be interfering with the correct reproduction of the original Pole figures?

**Comment by ralfHielscher** [comment]
sample symmetry is most likely triclinic, i.e., `1`. 622 is the crystal symmetry of Zircon.
and yes, this has a big impact on the pole figures :)

Ralf
